Explorer/Assets/Scripts/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -260,7 +260,7 @@ IMultiPool MultiPoolFactory() =>
             var equippedEmotes = new EquippedEmotes();
             var forceRender = new List<string>();
             var selfProfile = new SelfProfile(container.ProfileRepository, identityCache, equippedWearables, wearableCatalog,
-                emotesCache, equippedEmotes, forceRender, ParseSelfForcedEmotes(bootstrapContainer.ApplicationParametersParser));
+                emotesCache, equippedEmotes, forceRender, ParseSelfForcedEmotes(bootstrapContainer.ApplicationParametersParser), ParseDebugForcedEmotes(bootstrapContainer.DebugSettings.EmotesToAddToUserProfile));
```

Comment:
We could inject a single list by merging both, the ones from debug settings and the app params, with no need to modify the `SelfProfile` class. What do you think?

Reply:
Sounds good!